Content.Server/GameObjects/Components/Construction/MachineBoardComponent.cs

```diff
@@ -48,9 +75,9 @@ public void Examine(FormattedMessage message, bool inDetailsRange)
                 message.AddMarkup(Loc.GetString("[color=yellow]{0}x[/color] [color=green]{1}[/color]\n", amount, Loc.GetString(part.ToString())));
             }
 
-            foreach (var (material, amount) in MaterialRequirements)
+            foreach (var (material, amount) in MaterialIdRequirements)
             {
-                message.AddMarkup(Loc.GetString("[color=yellow]{0}x[/color] [color=green]{1}[/color]\n", amount, Loc.GetString(material.ToString())));
+                message.AddMarkup(Loc.GetString("[color=yellow]{0}x[/color] [color=green]{1}[/color]\n", amount, Loc.GetString(material)));
```

Comment:
As I said above, this should probably index the stack prototype and get the human-readable name

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/3387/commits/e8560e28a76adc8b9f6b32f0e28c8d58c580c00f